setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 
-VERSION = '0.1'
+VERSION = '0.2'
```

Comment:
Let's follow up before release that a bunch of the `setup.py` information references `Rubin Deliallisi`!

Reply:
Could we revert this and then @mhrabovcin 's work on a release process can bring this in?

Reply:
Bumping the version now is ok. @adamtheturtle @jr0d the release tooling PR - https://github.com/mesosphere/exhibitor-tls-artifacts-gen/pull/4